edx_solutions_api_integration/users/views.py

```diff
@@ -361,9 +363,15 @@ def get_queryset(self):
             if name:
                 queryset = queryset.filter(Q(profile__name=name) | Q(first_name=name) | Q(last_name=name))
 
+            if usernames:
+                queryset = queryset.filter(username__in=usernames)
+
             if email:
                 queryset = queryset.filter(email=email)
 
+            if emails:
+                queryset = queryset.filter(email__in=emails)
```

Comment:
I think it would be better to enhance the existing email parameter to support multiple emails rather than having two parameters for the same thing. Likewise rename userenames to username to be consistent.

Reply:
Also  the docs need to be updated.

Reply:
Done.